src/Pages/sunkyungnoh/Login/components/Appstore.scss

```
@@ -0,0 +1,24 @@
+.appStore {
+  width: 350px;
+  p {
+    font-size: 14px;
+    line-height: 18px;
+    margin: 10px 20px 10px 20px;
+    text-align: center;
+  }
+
+  .appStoreBtn {
+    display: flex;
+    justify-content: center;
+    margin: 10px 0 10px 0;
+  }
+
+  img {
+    width: auto;
+    height: 40px;
+  }
+
+  img:first-of-type {
+    margin-right: 8px;
```

```suggestion
  .appStoreBtn {
    display: flex;
    justify-content: center;
    margin: 10px 0 10px 0;
    
    img {
    width: auto;
    height: 40px;
    
    &:first-of-type {
    margin-right: 8px;
    }
  }
 }  
```
버튼안에 속해있는 이미지들이 조금 더 nesting 되어도 좋을 것 같습니다! 첫번째 이미지에는  &를 써서 넣을 수도 있겠네요!(아닐 수도 있어요^^)

---

감사합니다! 수정했습니다~!